### DISCUSSION TITLE
Add lime3ds core with vulkan

### DISCUSSION THREAD

**Discussion by Algiuxs** [Swordfish90/Lemuroid#960] [Ideas]
Add a lime3ds core and it's vulkan, opengl  renderer, it's like citra, but better and not dead.
Allow the user to switch between the lime3DS core and citra.

**Comment by x64x2** [comment]
![IMG_20240827_035024_456.jpg](https://github.com/user-attachments/assets/d6e3edbc-5ef9-4b2c-bce6-bfd56dbf4624)

**Reply by Algiuxs** [reply]
Less graphical glitches

**Comment by ghost** [comment]
I believe it is a solid reason, I had problems with Citra even though I had a good device.

**Comment by Algiuxs** [comment]
Lime3ds was renamed to azahar

**Reply by umar248** [reply]
Then should add azahar core with .CCI support for the emulator.
That would ve very helpful tbh
